.github/workflows/script-evaluation-dump.yml

```diff
@@ -33,3 +40,13 @@ jobs:
           S3_DUMP_DIR: s3://plutus/mainnet-script-dump/
           LOCAL_DUMP_DIR: ../mainnet-script-dump 
 
+      - name: Upload Dump
```

Comment:
Also what about this step? Do we want to upload right after dumping? 
@zliu41 is the `./scripts/upload-event-dump.sh` up-to-date?

Reply:
The one in `zliu41/dump-1.35.4` should be up to date.